docs/walkthrough/create-training-data.md

```diff
@@ -41,6 +24,19 @@ I'm (…) sorry!          apologize-english
 I'm sorry to have…      apologize-english
 Please, forgive me!     apologize-english
 ```
+
+When using image-to-image retrieval models, images can be represented as a path to a file:
```

Comment:
maybe say "When using image-to-image retrieval models, images can be represented as a URI or a path to a file:"

Since some of the examples below it are URLs, that makes a bit more sense.